src/main/charts/confluence/templates/configmap-fluentd.yaml

```diff
@@ -0,0 +1,40 @@
+{{ if .Values.fluentd.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "confluence.fullname" . }}-fluentd-config
+  labels:
+  {{- include "confluence.labels" . | nindent 4 }}
+data:
+  fluent.conf: |
```

Comment:
The below config should be taken very much as a first attempt. It's the simplest thing that'll work, and probably needs tweaking for prod usage.